.github/workflows/calibreapp-image-actions.yml

```diff
@@ -0,0 +1,22 @@
+name: Compress Images
+on:
+  pull_request:
+    paths:
+      - '**.jpg'
+      - '**.jpeg'
+      - '**.png'
+      - '**.webp'
+jobs:
+  build:
+    # Only run on Pull Requests within the same repository, and not from forks.
+    # if: github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
This is needed because of the limitations described in https://github.com/calibreapp/image-actions#process-pull-requests-from-forked-repositories

so https://github.com/calibreapp/image-actions#combined-workflow is probably the best

Reply:
@zerolab sure, I'll uncomment this, do you also want me to add cronjobs?

Reply:
https://github.com/calibreapp/image-actions#combined-workflow is the one that we want